packages/vuetify/src/components/VDataTable/MobileRow.ts

```diff
@@ -11,6 +11,7 @@ export default Vue.extend({
     headers: Array as PropType<DataTableHeader[]>,
     item: Object,
     rtl: Boolean,
+    hideDefaultHeader: Boolean,
```

Comment:
Can you please put `hideDefaultHeader` below `headers` to respect alphabetical order

Reply:
Leave this as a review. It let's me know that we are waiting on something and I can skip over it.

Reply:
@Kawacrepe Fixed

Reply:
Thanks you